grails-app/controllers/org/pih/warehouse/inventory/InventoryItemController.groovy

```diff
@@ -583,27 +583,8 @@ class InventoryItemController {
             return
         }
 
-        log.info("User chose to validate or there are errors")
-        Location warehouseInstance = Location.get(session?.warehouse?.id)
-
-        commandInstance.product = Product.load(commandInstance.product.id)
-        commandInstance.inventory = warehouseInstance?.inventory
-        commandInstance.inventoryLevel = InventoryLevel.findByProductAndInventory(commandInstance?.product, commandInstance?.inventory)
-
-        Product productInstance = commandInstance.product
-        List transactionEntryList = inventoryService.getTransactionEntriesByInventoryAndProduct(commandInstance?.inventory, [productInstance])
-
-        // Get the inventory warning level for the given product and inventory
-        commandInstance.inventoryLevel = InventoryLevel.findByProductAndInventory(commandInstance?.product, commandInstance?.inventory)
-
-        commandInstance.totalQuantity = inventoryService.getQuantityByProductMap(transactionEntryList)[productInstance] ?: 0
-
-        log.info "commandInstance.recordInventoryRows: "
-        commandInstance?.recordInventoryRows.each {
-            log.info "it ${it?.id}:${it?.lotNumber}:${it?.oldQuantity}:${it?.newQuantity}"
-        }
-
-        render(view: "showRecordInventory", model: [commandInstance: commandInstance])
+        flash.errors = commandInstance.errors
+        redirect(action: "showRecordInventory", params: ['product.id': commandInstance.product.id])
```

Comment:
I'm confused as to why passing `model: [commandInstance: commandInstance]` to the view isn't needed anymore. It looks like it's always used in the GSP (even when there are errors) but I'm never totally sure how those files are working so maybe I'm wrong. Is the fact that we're flashing errors bypass something?